python/lsst/pipe/drivers/utils.py

```diff
@@ -1,7 +1,15 @@
-from lsst.pipe.base import Struct
+from lsst.pipe.base import Struct, TaskRunner
 from lsst.pipe.tasks.coaddBase import CoaddDataIdContainer
 from lsst.pipe.tasks.selectImages import BaseSelectImagesTask, BaseExposureInfo
 
+class ButlerTaskRunner(TaskRunner):
```

Comment:
Consider adding this to pipe_base, as it might be more generally useful. That said, we have a lot of task runners that should probably go in pipe_base.